app/src/App.tsx

```diff
@@ -1,23 +1,24 @@
-import React, { useEffect, useState } from 'react';
-import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import React, {useEffect, useState} from 'react';
+import {BrowserRouter, Route, Routes} from 'react-router-dom';
 import './App.css';
 
 import {
-  getAuth,
-  User,
-  signInWithPopup,
-  GoogleAuthProvider,
-  sendSignInLinkToEmail,
-  isSignInWithEmailLink,
-  signInWithEmailLink,
-  signOut,
+    getAuth,
```

Comment:
my IDE automatically added this formatting in this file

Reply:
we can add some eslint rules later